src/java/org/apache/cassandra/service/ActiveRepairService.java

```diff
@@ -232,13 +231,13 @@ public void failSession(String session, boolean force)
     @Override
     public void setRepairSessionSpaceInMegabytes(int sizeInMegabytes)
     {
-        DatabaseDescriptor.setRepairSessionSpaceInMegabytes(sizeInMegabytes);
+        DatabaseDescriptor.setRepairSessionSpaceInMiB(sizeInMegabytes);
```

Comment:
nit: Might as well just do a find/replace over the whole codebase to see where we need to change `sizeInMegabytes` (or just `megabytes`) to `sizeInMebibytes`, etc.

Reply:
Good point, I was doing it about mb, kb, etc

Reply:
Looking into it then there are also public methods like `setRepairSessionSpaceInMegabytes` and `getRepairSessionSpaceInMegabytes`....`public` methods, I am hesitant here..

Reply:
If we change those we need to keep the old name as deprecated... I am not sure we really want to deal with this now here